halyard-web/src/main/java/com/netflix/spinnaker/halyard/controllers/v1/AccountController.java

```diff
@@ -140,6 +155,8 @@
     builder.setValidate(doValidate);
     builder.setRevert(() -> halconfigParser.undoChanges());
     builder.setSave(() -> halconfigParser.saveConfig());
+    Path stagingPath = halconfigDirectoryStructure.getConfigPath(deploymentName);
```

Comment:
Should this be named "configPath"? (Here and in a few other spots).

Reply:
If by a few, you mean 30. Yes ;)